torq.sh

```diff
@@ -18,6 +18,10 @@ if [ -z $QCON ]; then
   QCON="qcon"                                                                                       # set environment if not predefined
 fi
 
+if [ -z $QCMD ]; then                                                                               # set environment if not predefined
```

Comment:
Update this comment - it's not environment related. Also update the comment for QCON